### ISSUE TITLE
Help: how to get index/symbol from last_hidden, on text8?

### ISSUE THREAD

**Issue by moonblue333** [huggingface/pytorch-pretrained-BERT#274]
I am trying on text8 dataset.  I want to print next token.   The model in source code forward() output is loss, but I want to get logits and softmax result, and finally get next token in vocab.

how to get index/symbol from last_hidden, on text8?

Thanks

**Comment by thomwolf** [comment]
Hi,
There is no pretrained character-level model for text8 right now.
Only a word-level model trained on wikitext 103.
